### ISSUE TITLE
Add support for a comment box on each exception similar to the reference link

### ISSUE THREAD

**Issue by corneliutusnea** [exceptionless/Exceptionless#131]
I'd like to see a comment box on the exception so I can add some comments on the exception.
The reference link is nice but very limited.

**Comment by niemyjski** [comment]
What all would you want to comment on about? I like the idea of having system level notes and maybe a deobfuscated stack trace. I think we need notes, but I also want to keep it simple (not a full fledged issue tracker like lira (do one thing really well :)).

**Comment by corneliutusnea** [comment]
Well, stuff like: "We've seen this issue before on couple of other scenarios. I'll have [personx] investigate before we raise this ..." or "I'm investigating this .." which is enough to not have another engineer look into it.

Generally comments are good to help people on track especially when you work in multiple offices and multiple people monitoring the reports.

**Comment by niemyjski** [comment]
Yeah, I get that. Internally we've used our promote button (web hook via zapper) to automatically create a trello card. It would then add a reference link and when the card was moved to the done lane the stack would be marked as fixed).

**Comment by markusgud** [comment]
I see this as an excellent way to specify for example in what scenarios this error happened and how to reproduce it. Especially for not as critical bugs that one wants to work on later.

Otherwise it is easy to forget how to reproduce the bug based on the exception stack (there could be a faulty data insert done before a redirect or reload is performed).

**Comment by StefanKoell** [comment]
+1. Notes for stacks would be very useful.
